package.json

```diff
@@ -36,16 +36,18 @@
     "format": "prettier --ignore-path .gitignore --write .",
     "format-check": "prettier --ignore-path .gitignore --check .",
     "lint": "eslint --ignore-path .gitignore --ext .js,.tsx .",
+    "watch": "npm run build -- --watch",
     "type-check": "tsc --noEmit"
   },
   "dependencies": {
-    "@storybook/addons": "^6.0.21",
-    "@storybook/api": "^6.0.21",
-    "@storybook/components": "^6.0.21",
-    "@storybook/theming": "^6.0.21",
+    "@storybook/addons": "^6.1.10",
+    "@storybook/api": "^6.1.10",
+    "@storybook/components": "^6.1.10",
+    "@storybook/theming": "^6.1.10",
     "core-js": "^3.6.5",
     "memoizerific": "^1.11.3",
-    "react": "^16.13.1"
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1"
```

Comment:
Is `react-dom` needed? I wasn't a dependency before.

Reply:
hey @rbardini, yeah sort of. I think this comes as part of https://github.com/storybookjs/storybook/pull/13195 SB 6.1 and above. But checking the add-on backgrounds the use it as a peer dependency https://github.com/storybookjs/storybook/blob/next/addons/backgrounds/package.json. we can do the same.

Reply:
I think that would be better, for both React dependencies.

The range they use (`^16.8.0 || ^17.0.0`) is also more inclusive, we can copy that as well 🙂